app/components/account/TokenExtensionsCard.tsx

```diff
@@ -0,0 +1,283 @@
+            break;
+        }
+        case 'defaultAccountState': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('default-account-state')),
+                name: 'Default Account State',
+                status: 'active',
+            };
+            break;
+        }
+        case 'immutableOwner': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('immutable-owner')),
+                name: 'Immutable Owner',
+                status: 'active',
+            };
+            break;
+        }
+        case 'memoTransfer': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('required-memo')),
+                name: 'Required Memo',
+                status: 'active',
+            };
+            break;
+        }
+        case 'nonTransferable': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('non-transferable-token')),
+                name: 'Non-Transferable Token',
+                status: 'active',
+            };
+            break;
+        }
+        case 'nonTransferableAccount': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('non-transferable-token')),
+                name: "Non-Transferable Token's Account",
+                status: 'active',
+            };
+            break;
+        }
+        case 'cpiGuard': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('cpi-guard')),
+                name: 'CPI Guard',
+                status: 'active',
+            };
+            break;
+        }
+        case 'permanentDelegate': {
+            return {
+                description:
+                    'Delegates permanent authority to a specific address that can transfer or burn tokens from any account holding this token, providing centralized administrative control over the token ecosystem.',
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('permanent-delegate')),
+                name: 'Permanent Delegate',
+                status: 'active',
+                tooltip:
+                    'Delegates permanent authority to a specific address that can transfer or burn tokens from any account holding this token, providing centralized administrative control over the token ecosystem.',
+            };
+            break;
+        }
+        case 'transferHook': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('transfer-hook')),
+                name: 'Transfer Hook',
+                status: 'active',
+            };
+            break;
+        }
+        case 'transferHookAccount': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('transfer-hook')),
```

Comment:
description: "List of accounts that the token program requires to execute custom instruction logic"

Reply:
fixed